database/RocksIterator.java

```diff
@@ -44,6 +45,8 @@
     private boolean isClosed;
     org.rocksdb.RocksIterator internalRocksIterator;
 
+    public static Stats stats = new Stats();
```

Comment:
i had a chat with ganesh and we think we should delete these objects and usages for now until we can implement it properly with per-query context to log the statistics against a particular query plan with